wallet/src/adapters/keybase.rs

```diff
@@ -242,6 +281,11 @@ impl WalletCommAdapter for KeybaseWalletCommAdapter {
 							// Reply to the same channel with topic SLATE_SIGNED
 							Ok(_) => match send(slate, channel, SLATE_SIGNED, TTL) {
 								true => {
+									notify_on_receive(
+										config.keybase_notify_ttl,
+										channel.to_string(),
+										tx_uuid.to_string(),
+									);
 									println!("Returned slate to {}", channel);
```

Comment:
nit: put the println before the notification in case the notification fails but the slate is still returned

Reply:
It would be interesting to replace all `println!` by standard logging macros: `debug!`, `info!`, etc.

Reply:
both accepted and updated, thanks for the review.